modules/ROOT/pages/managing-users.adoc

```diff
@@ -105,6 +106,28 @@ Verify that your identity configuration is correct before creating the identity
 
 If your identity provider supports uploading service-provider metadata, you can click the *Anypoint service provider metadata* link to download your SAML configuration settings in an XML file. You can then upload this file to your identity provider to configure Anypoint Platform. Note that you must have a SAML 2.0 identity configuration to export this data.
 
+
+== SSO Redirects using Cookies
+
+Anypoint Platform uses cookies to ensure SSO users are redirected to the same location from which they initiated their login.
```

Comment:
```suggestion
Anypoint Platform uses cookies to ensure that SSO users are redirected to the same location from which they initiated their login.
```
See "ensure, ensure that" in style ref.

Reply:
Do you think it would be helpful to introduce the following sentences as a numbered sequence of actions (ordered list) that occur to ensure that users are eventually left where they started? Because this text seems to focus on PROCESS.
It could even be illustrated by a numbered graphic, but I suspect it is not worth the effort.

Reply:
Know that this is just an idea for you to consider, not better than what you have here, which is perfectly fine.